app/views/ops/_settings_server_tab.html.haml

```diff
@@ -205,7 +205,7 @@
         = _("SSL Verify Mode")
       .col-md-8
         = select_tag('smtp_openssl_verify_mode',
-                      options_for_select(GenericMailer.openssl_verify_modes.collect { |m| [m.titleize, m] }, @edit[:new][:smtp][:openssl_verify_mode]),
+                      options_for_select(GenericMailer.openssl_verify_modes.collect { |m| [_(m.titleize), m] }, @edit[:new][:smtp][:openssl_verify_mode]),
```

Comment:
This won't really work, since `m.titleize` values never made it into the catalog, right?